### PR TITLE
Update meta.yaml to match requirements.txt

### PR DESCRIPTION
**Pull request by rxm7706**
removed old dependencies termcolor
reordered to match grayskull ouput

Review - removed
    - dataclasses
    - typing_extensions >=3.10.0.0

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,39 +20,38 @@ requirements:
...
     - requests >=2.20
     - ruamel.yaml >=0.16,<0.17.18
     - scipy >=1.6.0
-    - termcolor >=1.1.0,<2.1.0
     - tqdm >=4.59.0
+    - typing-extensions >=3.10.0.0
     - tzlocal >=1.2
-    - typing_extensions >=3.10.0.0
+    - urllib3 >=1.26
+    #- dataclasses
```
Comment (xhochy) [inline recipe/meta.yaml:53]: If they are no longer required, you should delete the line.

**Review by xhochy — CHANGES_REQUESTED** [review]
I'm confused the grayskull automation didn't do this.

File: recipe/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipe/meta.yaml:53]: Hi @xhochy - 
That's correct, I didnt find dataclasses in either the upstream requirements nor was it generated by grayskull, but was wondering if it was required for older versions of python.. 
Same for  typing-extensions >=3.10.0.0, I wasnt sure if there was a difference between the - and _ packages. 
    - typing-extensions &  typing_extensions
 I had those two packages dataclasses and typing_extensions called out in the title of this issue, hoping an extra pair of eyes, review from someone. 

If the recipe looks right, I will remove both dataclasses and  typing_extensions lines. 
Thank you, Regards

Comment (xhochy) [reply recipe/meta.yaml:53]: There is no difference between both packages. One is an alias for the other. This is all looking good.

**Comment by rxm7706** [comment]
@xhochy / maintainers ready for review / merge

**Review by xhochy — APPROVED** [review]

**Pull request merged**